<?xml version="1.0"?>
<sdf version="1.6">
  <model name="payload_box">
    <static>false</static>
    <link name="link">
      <gravity>true</gravity>
      <pose>0 0 0.05 0 0 0</pose>

      <inertial>
        <mass>0.2</mass>
        <inertia>
          <ixx>0.001</ixx><iyy>0.001</iyy><izz>0.001</izz>
        </inertia>
      </inertial>

      <collision name="collision">
        <geometry>
          <box><size>0.12 0.12 0.12</size></box>
        </geometry>
      </collision>

      <visual name="visual">
        <geometry>
          <box><size>0.12 0.12 0.12</size></box>
        </geometry>
        <material>
          <ambient>1 0.5 0 1</ambient>
          <diffuse>1 0.5 0 1</diffuse>
        </material>
      </visual>
    </link>
  </model>
</sdf>

